independent-publisher-connectors/Brønnøysundregistrene/apiProperties.json

```diff
@@ -1,9 +1,9 @@
 {
   "properties": {
     "connectionParameters": {},
-    "iconBrandColor": "#0B576F",
+    "iconBrandColor": "#da3b01",
     "capabilities": [],
     "publisher": "Ahmad Najjar",
-    "stackOwner": "Br�nn�ysundregistrene (Br�nn�ysund registries)"
+    "stackOwner": "Brønnøysundregistrene (Brønnøysund registries)"
```

Comment:
@vbossica we don't allow any special characters. can you please revert this change? Thank you

Reply:
Replaced the `ø` with `o` in the last commit